internal/core/service/project/service.go

```diff
@@ -214,7 +215,7 @@ func (s *Storage) ListDonationByProjectId(ctx context.Context, input model.ListP
 	}
 
 	// jika status project masih need_review, maka tidak bisa donate
-	if p.Status == _type.PROJECT_NEED_REVIEW {
+	if p.Status == _type.PROJECT_NEED_REVIEW && ctx.Value("isAdmin") == false {
```

Comment:
![Screenshot from 2024-06-02 16-02-30](https://github.com/nabitu/donation-hub/assets/12779618/f7ad8d9d-337c-4f39-86f9-2802c8af22e8)
I tried without logging in, and still managed to get data that shouldn't be seen.